scripts/block-sync.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+# Start a consensus node and block sync from genesis to tip for Celestia networks (Arabica, Mocha, Mainnet)
+# Usage: ./scripts/block-sync.sh --network <arabica|mocha|mainnet>
+
+set -o errexit
+set -o nounset
+
+# Default values
+NODE_NAME="${NODE_NAME:-node-name}"
+
+usage() {
+  echo "Usage: $0 --network <arabica|mocha|mainnet>"
```

Comment:
[optional] and only encouraged if we do this in the state sync PR

```suggestion
  echo "Usage: $0 --network <Arabica | Mocha | Mainnet>"
```